src/script.js

```diff
@@ -30,4 +30,9 @@ addEventListener("submit", (event) => {
```

Comment:
Great job on adding login functionality to the script.js file. It's looking good! Just one suggestion: instead of hardcoding the user credentials, consider implementing a more secure authentication system.